packages/newsroom-manager/src/Newsroom.tsx

```diff
@@ -166,10 +187,17 @@ class NewsroomComponent extends React.Component<NewsroomProps & DispatchProp<any
   };
 
   public isStepDisabled = (index: number) => {
```

Comment:
@walfly I updated this function to disable steps based on current user's role, but then saw that this function isn't actually in use. Before I go digging, should this be removed, or is there another way we're handling this, or is this functionality in flux as you work on the manager steps, or should I go ahead and hook it into steps as they are now?